cmd/tigerblood/main.go

```diff
@@ -4,11 +4,23 @@ import (
 	"fmt"
 	"github.com/DataDog/datadog-go/statsd"
 	"go.mozilla.org/tigerblood"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
 	"log"
 	"net/http"
 	"os"
 )
 
+func loadCredentials() (map[string]string, error) {
+	creds := make(map[string]string)
```

Comment:
We may need more configuration in the future, but credentials.yml is intended to be encrypted, whereas other config files may not need to be encrypted.